src/include/daos/mgmt.h

```diff
@@ -65,6 +67,7 @@ struct dc_mgmt_sys {
 	struct sys_info		sy_info;
 };
 
+int dc_network_cfg(const char *name);
```

Comment:
For consistency with other functions here, seems like this should be named something like `dc_mgmt_net_cfg()`.

Reply:
Sure, I can do that.